src/generic.rs

```diff
@@ -67,19 +67,18 @@ where
     T: Beef + ?Sized,
     U: Capacity,
 {
-    /// Borrowed data.
-    ///
-    /// # Example
-    ///
-    /// ```rust
-    /// use beef::Cow;
-    ///
-    /// let borrowed: Cow<str> = Cow::borrowed("I'm just a borrow");
-    /// ```
-    #[inline]
-    pub fn borrowed(val: &'a T) -> Self {
-        let (ptr, fat, cap) = T::ref_into_parts::<U>(val);
+    #[cfg(feature = "const_deref")]
+    const fn borrowed_from_parts(ptr: NonNull<T::PointerT>, fat: usize, cap: U::Field) -> Self {
+        Cow {
+            ptr,
+            fat,
+            cap,
+            marker: PhantomData,
+        }
+    }
 
+    #[cfg(not(feature = "const_deref"))]
+    fn borrowed_from_parts(ptr: NonNull<T::PointerT>, fat: usize, cap: U::Field) -> Self {
```

Comment:
```suggestion
    const fn borrowed_from_parts(ptr: NonNull<T::PointerT>, fat: usize, cap: U::Field) -> Self {
```

These impls are the same, so no need to split them based on the cfg.

Reply:
> These impls are the same, so no need to split them based on the cfg.


But `borrowed_from_parts` require unstable const `slice_from_raw_parts` :(

Reply:
To call it, maybe, but not to define it; the definition is just `Cow { ptr, fat, cap, marker }`, which is `const`.

Unless I'm blind and missing something obvious.